Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Tags-vpost-7.0.md

```diff
@@ -0,0 +1,77 @@
+---
+keywords: tags property editors v7 version7
+versionFrom: 7.0.0
```

Comment:
How sure are you about the versionFrom?

Reply:
Gut feeling to be honest - I can try and setup some 7 and 7.3 to see if there are any differences between the version though. I think that should be sufficient, no? Or we can wait and see if anyone reports issues in the forum if/when they come across it?

Reply:
Yup, agree. Just forgot how much has really happened since v7.0 was fresh and shiny! :) - It's been updated now.